backend/label/tests.py

```diff
@@ -1 +1,89 @@
 # Create your tests here.
+from django.contrib.auth.models import User
+from rest_framework import status
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+
+from project.models import Project
+from label.models import Label
+from repository.models import Repository
+
+
+class LabelTests(APITestCase):
+
+    empty_project_pk = None
+    non_empty_project_pk = None
+    no_project_pk = 123456
+
+    def setUp(self) -> None:
+        admin = User.objects.create_superuser(username="admin", password="admin")
+        repository1 = Repository.objects.create(url='https://github.com/Tim6FTN/UKS', name='UKS')
+        repository2 = Repository.objects.create(url='https://github.com/Tim6FTN/UKS', name='UKS')
```

Comment:
Shouldn't this raise an exception as repository URL's are unique?

Reply:
Nope, unique constraint was removed from the model